src/components/Loading.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import ReactLoading from 'react-loading';
+import colors from '../index.scss';
+
+
+const Loading = (props) => {
```

Comment:
I think we've been using 2 spaces of indentation in js. Maybe we should keep it in all files, what do you think? The linter would help solving this kind of problems?